### PR TITLE
Add fdrs fetched date

### PR DESCRIPTION
**Pull request by k9845**
Addresses
- https://github.com/IFRCGo/go-api/issues/2118

## Changes
 - Added field `fdrs_data_fetched_year`

# NOTE: Please run the following command on staging server
`docker-compose exec serve bash  ./manage.py ingest_databank`

## Checklist
Things that should succeed before merging.

- [ ] Updated/ran unit tests
- [ ] Updated CHANGELOG.md

## Release

If there is a version update, make sure to tag the repository with the latest version.

### CODE REVIEW

File: databank/models.py [hunk 1]
```diff
@@ -535,6 +535,11 @@ class CountryOverview(models.Model):
         verbose_name=_('HDR GII'),
         null=True, blank=True
     )
+    fdrs_data_fetched_year = models.CharField(
+        verbose_name=_('FDRS Data Fetched Year'),
+        null=True, blank=True,
+        max_length=50
+    )
```
Comment (thenav56) [inline databank/models.py:542]: Any reason why we are storing year in string?

File: databank/management/commands/sources/FDRS.py [hunk 2]
```diff
@@ -120,11 +121,15 @@ def load(country, overview, fdrs_data):
     if country.iso is None or fdrs_data is None:
         return
 
+    get_max_year = lambda data: max(int(item['year']) for item in fdrs_data.values())
+    fdrs_data_fetched_year = get_max_year(fdrs_data)
```
Comment (thenav56) [inline databank/management/commands/sources/FDRS.py:125]: Any reason why we need lambda here instead of simple statement?

**Review by thenav56 — CHANGES_REQUESTED** [review]

File: databank/models.py (hunk 1 above)
Comment (k9845) [reply databank/models.py:542]: We get year as string from data source and populating that here.

File: databank/management/commands/sources/FDRS.py (hunk 2 above)
Comment (k9845) [reply databank/management/commands/sources/FDRS.py:125]: Just creating this to look clean.

Comment (k9845) [reply databank/management/commands/sources/FDRS.py]: Replaced with normal statement

**Review by samshara — APPROVED** [review]

**Pull request merged**